src/plugins/presentation_util/public/components/solution_toolbar/solution_toolbar.tsx

```diff
@@ -8,12 +8,13 @@
 
 import React, { ReactElement } from 'react';
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { SolutionToolbarButton } from 'src/plugins/shared_ux/public';
```

Comment:
I don't think this is going to work.  Only Typescript types can be imported without relative path, unless something has changed recently, (@spalger ?)

Reply:
Correct, use the relative path please